pkg/cacheutil/memcached_client.go

```diff
@@ -244,6 +276,18 @@ func newMemcachedClient(
 	c.duration.WithLabelValues(opGetMulti)
 	c.duration.WithLabelValues(opSet)
 
+	c.valueSize = promauto.With(reg).NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
Now that I think maybe I should have split this metric. It could be confusing because for set operation it only tracks one item, for get it track multiple. What do you think?

Reply:
I personally think it's fine having the operation as label. It's consistent with other metrics. It's the size of the data of a single operation/call.